openlibrary/plugins/openlibrary/js/readmore.js

```diff
@@ -1,18 +1,18 @@
 export function initReadMoreButton() {
-    $('.read-more-button').on('click',function(){
+    $('.read-more-button').on('click', function () {
```

Comment:
```suggestion
    $('.read-more-button').on('click',function(){
```
Let's not change any unrelated code.